options/gorm.proto

```diff
@@ -40,8 +40,41 @@ extend google.protobuf.FieldOptions {
 }
 
 message GormFieldOptions {
-  optional string tags = 1;
-  optional bool drop = 2;
+    optional GormTag tag = 1;
+    optional bool drop = 2;
+    optional HasOneOptions has_one = 3;
+    optional HasManyOptions has_many = 4;
+}
+
+message GormTag {
+    optional string column = 1;
+    optional string type = 2;
+    optional int32 size = 3;
+    optional int32 precision = 4;
+    optional bool primary_key = 5;
+    optional bool unique = 6;
+    optional string default = 7;
+    optional bool not_null = 8;
+    optional bool auto_increment = 9;
+    optional string index = 10;
+    optional string unique_index = 11;
```

Comment:
Some of these options are used like DDL (index, auto_increment, default), since we're advocating manually written DDL should we keep these fields or not even include them?

Reply:
If somebody's database schema is simple enough in order to build it with GORM, why we should forbid this?

Reply:
That's fair. My concern would be that gorm has limited migration support, so if at some point the schema has to change beyond what gorm can handle like this, it would be an awkward half-gorm/half-manually written schema. I suppose that would be 'their problem' though.

Reply:
Yeah, sure. Indeed, GORM's migration support is safe: it doesn't remove disappeared entities(columns, tables), but only creates new ones. Maybe that approach is fine if you have small databases and don't change your schema frequently. Also this can be considered as an advantage e.g. if you want to keep several versions of application working on the same database.